### ISSUE TITLE
Automation failure : test_subscribe_system_to_cv 

### ISSUE THREAD

**Issue by tkolhar** [SatelliteQE/robottelo#2286]
TestNames:
tests.foreman.api.test_multiple_paths.EntityReadTestCase.test_entity_read_21_System

Error Message
'organization_id'
Stacktrace

  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/2e7576f6/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-api/tests/foreman/api/test_multiple_paths.py", line 696, in test_entity_read
    self.assertIsInstance(entity_cls(id=entity_id).read(), entity_cls)
  File "/home/jenkins/shiningpanda/jobs/2e7576f6/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entities.py", line 3181, in read
    organization_id = attrs.pop('organization_id')

**Comment by tkolhar** [comment]
when executed locally using nosetests : 

]$ nosetests -c robottelo.properties test_entity_read_21_System /home/tkolhar/git/robottelo/tests/foreman/api/test_multiple_paths.py
ERROR: @Test: Issue a POST request and check the returned status code.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/tkolhar/git/robottelo/tests/foreman/api/test_multiple_paths.py", line 239, in test_post_status_code
    skip_if_sam(self, entity_cls)
  File "/home/tkolhar/git/robottelo/tests/foreman/api/test_multiple_paths.py", line 98, in skip_if_sam

will investigate some more and update the issue
    in entity()._meta['server_modes']  # pylint:disable=protected-access
AttributeError: 'Environment' object has no attribute '_meta'

**Comment by Ichimonji10** [comment]
@tkolhar, have you updated to the most recent version of nailgun? It looks like you're using an outdated version. Make sure to execute `pip install --upgrade nailgun` and `pip install --upgrade requests`.

**Comment by Ichimonji10** [comment]
This bug is due to the set of data being returned by the server changing. Fixing NailGun so that it accepts whatever the server returns is not too hard. A slightly harder but necessary second step is figuring out if this change has been made only in certain software versions. For example, has this change been made only on Sat but not Foreman?

**Comment by Ichimonji10** [comment]
*sigh*

The API broke yet again. A system can be associated with an organization, but HTTP GET requests to `/katello/api/v2/systems/:id` do not reflect that fact. See:

```python
>>> attrs = entities.System(
...     id='8485c939-5c4d-4172-8131-e9e375e1c21b'
... ).read_json()
>>> 'organization' in attrs
False
>>> 'organization_id' in attrs
False
>>> l = list(attrs.keys())
>>> l.sort()
>>> l
[
    'activation_keys',
    'autoheal',
    'checkin_time',
    'content_overrides',
    'content_view',
    'content_view_id',
    'created',
    'customInfo',
    'description',
    'distribution',
    'entitlementStatus',
    'environment',
    'errata_counts',
    'host',
    'hostCollections',
    'id',
    'installedProducts',
    'katello_agent_installed',
    'katello_id',
    'location',
    'name',
    'permissions',
    'products',
    'registered_by',
    'release',
    'release_ver',
    'service_level',
    'uuid',
]
```

**Comment by Ichimonji10** [comment]
I'll file a bug and a PR to Robottelo to fix this test failure.

**Comment by Ichimonji10** [comment]
Oh, wait. I already filed [BZ 1123494](https://bugzilla.redhat.com/show_bug.cgi?id=1223494) and the bug won't be fixed until Sat 6.2.0. I might be able to compensate for this bug in 6.1.0 by issuing a no-op HTTP PUT request to `/katello/api/v2/systems/:id` and fetching the organization ID from the response.
